lib/taurus/cli/common.py

```diff
@@ -0,0 +1,97 @@
+#############################################################################
+##
+# This file is part of Taurus
+##
+# http://taurus-scada.org
+##
+# Copyright 2011 CELLS / ALBA Synchrotron, Bellaterra, Spain
+##
+# Taurus is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+##
+# Taurus is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+##
+# You should have received a copy of the GNU Lesser General Public License
+# along with Taurus.  If not, see <http://www.gnu.org/licenses/>.
+##
+#############################################################################
+
```

Comment:
It would be nice if a module docstring is added describing what this module provides and how it is expected to be used (e.g. putting a simple code example).

Also, It would be nice to mention this submodule in the docstring of `taurus.cli` (in the `__init__.py`)

Ideally (but if this is too much, it could be created as a separate feature request) a small guide should be added to the docs (e.g. in `taurus/doc/source/devel/cli_guide.srt`)

Reply:
I've added docstring and updated docstring of `__init__.py`. Also, I've added two examples